src/components/AddressSearch.js

```diff
@@ -263,6 +263,7 @@ const AddressSearch = (props) => {
                             displayListViewBorder && styles.borderTopRounded,
                             displayListViewBorder && styles.borderBottomRounded,
                             displayListViewBorder && styles.mt1,
+                            displayListViewBorder && styles.pv1,
```

Comment:
`pv1` uses a vertical padding of 4 pixels. the focus ring is slightly cut (not much visible). Wouldn't `{paddingVertical: 6}` be better here?

Reply:
I think we should create a `StyleUtils` helper that takes `shouldDisplayListViewBorder` as a parameter.

Reply:
I think `shouldDisplayBorder` is better.